edsnlp/pipes/trainable/span_classifier/span_classifier.py

```diff
@@ -480,6 +516,9 @@ def forward(self, batch: SpanClassifierBatchInput) -> BatchOutput:
                         binding_scores[:, bindings_indexer],
                         batch["targets"][:, group_idx],
                         reduction="sum",
+                        weight=torch.tensor(self.label_weights)[bindings_indexer].to(
+                            binding_scores.device
```

Comment:
Do you have some results demonstrating an improvement in accuracy when weighting the loss ?